v0/destinations/posthog/transform.js

```diff
@@ -74,6 +74,21 @@ const responseBuilderSimple = (message, category, destination) => {
     ...payload.properties
   };
 
+  // Convert the distinct_id to string as that is the needed type in destinations.
+  if (!(payload.distinct_id === null || payload.distinct_id === undefined)) {
```

Comment:
Let's use the method `isDefinedAndNotNull` from https://github.com/rudderlabs/rudder-transformer/blob/738121e00997ebc438cc41318781f8a9b5ab53fb/v0/util/index.js#L30

Reply:
fixed as suggested